build.sbt

```diff
@@ -8,13 +8,12 @@ scalaVersion := "2.13.10"
 val circeVersion = "0.14.1"
 val awsVersion = "2.20.162"
 val zioVersion = "1.0.14"
-val jacksonVersion = "2.14.1"
```

Comment:
The jackson version overrides were for security vulnerabilities but are no longer necessary - we now get a more recent version of jackson via our dependencies